yang/ietf-l3vpn-ntw.yang

```diff
@@ -872,75 +849,11 @@ module ietf-l3vpn-ntw {
                           "CVLAN identifier.";
                       }
                     }
-                    container qinany {
-                      when "derived-from-or-self(../type, "
-                         + "'vpn-common:qinany')" {
-                        description
-                          "Only applies when the type of the
-                           tagged interface is 'qinany'.";
-                      }
-                      if-feature "vpn-common:qinany";
-                      description
-                        "Container for QinAny.";
-                      leaf tag-type {
-                        type identityref {
-                          base vpn-common:tag-type;
-                        }
-                        default "vpn-common:s-vlan";
-                        description
-                          "Tag type. By default, the tag type is
-                           's-vlan'.";
-                      }
-                      leaf svlan-id {
-                        type uint16;
-                        mandatory true;
-                        description
-                          "Service VLAN ID.";
-                      }
-                    }
-                    container vxlan {
-                      when "derived-from-or-self(../type, "
-                         + "'vpn-common:vxlan')" {
-                        description
-                          "Only applies when the type of the
-                           tagged interface is 'vxlan'.";
-                      }
-                      if-feature "vpn-common:vxlan";
-                      description
-                        "QinQ.";
-                      leaf vni-id {
-                        type uint32;
-                        mandatory true;
-                        description
-                          "VXLAN Network Identifier (VNI).";
-                      }
-                      leaf peer-mode {
-                        type identityref {
-                          base vpn-common:vxlan-peer-mode;
-                        }
-                        default "vpn-common:static-mode";
-                        description
-                          "Specifies the VXLAN access mode. By default,
-                           the peer mode is set to 'static-mode'.";
-                      }
-                      leaf-list peer-ip-address {
-                        type inet:ip-address;
-                        description
-                          "List of peer's IP addresses.";
-                      }
-                    }
                   }
-                  container bearer {
+                  container remote-endpoints {
```

Comment:
The proposed name is confusing. 

This is more a l2 termination. We may call it transport-logical-interface or something similar.